modules/ivc_champva/app/controllers/ivc_champva/v1/pega_controller.rb

```diff
@@ -66,7 +66,13 @@ def send_email(form, file_count)
             created_at: form.created_at.strftime('%B %d, %Y')
           }
 
-        IvcChampva::Email.new(form_data).send_email
+        ActiveRecord::Base.transaction do
+          if IvcChampva::Email.new(form_data).send_email
+            update_email_sent(form_uuid)
+          else
+            raise ActiveRecord::Rollback, 'Pega Status Update Email send failure'
+          end
+        end
       end
```

Comment:
Wrapped logic in transaction to avoid updating the DB if the email send is a failure